src/main/java/com/exasol/exasoltestsetup/ExasolTestSetup.java

```diff
@@ -65,8 +67,9 @@ public interface ExasolTestSetup extends AutoCloseable {
      * @param serviceAddress service address in the format host:port
      * @return modified service address
      */
-    public default ServiceAddress makeTcpServiceAccessibleFromDatabase(final ServiceAddress serviceAddress) {
-        if (serviceAddress.isLocal()) {
+    public default InetSocketAddress makeTcpServiceAccessibleFromDatabase(final InetSocketAddress serviceAddress) {
+        final boolean isLocalAddress = Set.of("localhost", "127.0.0.1").contains(serviceAddress.getHostName());
```

Comment:
This isn't supporting IP V6, but good enough here.

Reply:
IPv6 was not supported before. If we can add later it if required.